### PR TITLE
feat: Set Gsm Signal Strength

### LINKED ISSUE #367: Add GSM signal endpoint
## Description
Add GDM signal endpoint to set GSM signal strength (Emulator only)

## Endpoint Documentation
http://appium.io/docs/en/commands/device/network/gsm-signal/

### PR DESCRIPTION
**Pull request by akinsolb**
closes #367

## Change list

Please provide briefly described change list which are you going to propose. 
 
## Types of changes

What types of changes are you proposing/introducing to .NET client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] New feature (non-breaking change which adds value to the project)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Documentation
- [x] Have you proposed a file change/ PR with appium to update documentation? 
## Integration tests
- [x] Have you provided integration tests to pass against the beta version of appium? (for Bugfix or New feature)

## Details
Add Android API for GSM Signal (Emulator only) command 

Please provide more details about changes if it is necessary. If there are new features you can provide code samples which show the way they
work and possible use cases. Also you can create [gists](https://gist.github.com) with pasted C# code samples or put them here using markdown. 
About markdown please read [Mastering markdown](https://guides.github.com/features/mastering-markdown/) and [Writing on GitHub](https://help.github.com/categories/writing-on-github/)

### CODE REVIEW

File: src/Appium.Net/Appium/Android/AndroidCommandExecutionHelper.cs [hunk 1]
```diff
@@ -117,6 +117,9 @@ public static ConnectionType GetConection(IExecuteMethod executeMethod)
                 PrepareArguments(new[] {"phoneNumber", "action"},
                     new object[] {number, gsmCallAction.ToString().ToLowerInvariant()}));
 
+        public static void SetGsmStrength(IExecuteMethod executeMethod, GsmSignalStrength gsmSignalStrength) =>
+            executeMethod.Execute(AppiumDriverCommand.SetGsmSignalStrength, PrepareArgument("signalStrength", gsmSignalStrength));
```
Comment (mykola-mokhnach) [inline src/Appium.Net/Appium/Android/AndroidCommandExecutionHelper.cs:121]: is the enum value automatically converted to string when passed to the request body?

Comment (akinsolb) [reply src/Appium.Net/Appium/Android/AndroidCommandExecutionHelper.cs:121]: From the top of my memory while testing, it passes the enum index value

Comment (akinsolb) [reply src/Appium.Net/Appium/Android/AndroidCommandExecutionHelper.cs:121]: I'l double check now

Comment (akinsolb) [reply src/Appium.Net/Appium/Android/AndroidCommandExecutionHelper.cs:121]: Yeah it passes the enum value, like so:
`{"signalStrength": 2}`
Prefer the string?

Comment (mykola-mokhnach) [reply src/Appium.Net/Appium/Android/AndroidCommandExecutionHelper.cs:121]: I prefer it to be more explicit, but not sure how explicit this is for C# in particular. Leaving it up to you

**Review by mykola-mokhnach — APPROVED** [review]

File: src/Appium.Net/Appium/Android/AndroidCommandExecutionHelper.cs (hunk 1 above)
Comment (akinsolb) [reply src/Appium.Net/Appium/Android/AndroidCommandExecutionHelper.cs:121]: I'm happy with the enum value

**Pull request merged**